src/render.js

```diff
@@ -35,13 +35,21 @@ const validateJson = (input) => {
 };
 
 // helper function to switch between different rendering engines/formats
-const engine = async (input, { template = null, output = null } = {}) => {
+const engine = async (
+  input = null,
+  { template = null, output = null, file = null } = {}
+) => {
   validateOptions({ template: template, output: output });
+
+  if (file !== null && fs.existsSync(file)) {
+    input = fs.readFileSync(file);
+  }
```

Comment:
We need to catch the case in which the file does not exist and print a human-friendly error from our app : )

Reply:
That's true! Going to fix that! xD